composer.json

```diff
@@ -8,10 +8,30 @@
             "email": "changeme@mailinator.com"
         }
     ],
-    "autoload" : {
-        "classmap": ["."]
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/pear/Text_Diff",
+            "no-api": true
+        }
+    ],
+    "autoload": {
+        "classmap": [
+            "class.csstidy_optimise.php",
+            "class.csstidy_print.php",
+            "class.csstidy.php"
+        ]
+    },
+    "autoload-dev": {
+        "classmap": [
+            "testing"
+        ]
     },
     "require": {
 
+    },
+    "require-dev": {
+        "pear/text_diff": "^1.2",
+        "simpletest/simpletest": "^1.1"
```

Comment:
I initially tried locking these at the versions mentioned in the `unit-tests.php` file, but simpletest didn't publish the `1.0.x` version on github at all (and thus is available in packagist).  Using `1.1` of simpletest did require a couple of small changes to make the tests work.